Kernel/Devices/BlockDevice.cpp

```diff
@@ -30,7 +30,7 @@ BlockDevice::~BlockDevice()
 
 bool BlockDevice::read_block(u64 index, UserOrKernelBuffer& buffer)
```

Comment:
optional commit title nitpick: `Fix` => `Remove` it's just a bit more descriptive